.github/workflows/pa11y.yml

```diff
@@ -8,8 +8,8 @@ name: pa11y tests
 on: [pull_request]
 
 jobs:
-   build_and_test:
-    name: Build and test
+   pa11y_scan:
```

Comment:
`build_and_test` had been used for both this and the `tests` workflow. Led to weird behaviors with required checks.